tools/export/sw4stm32/__init__.py

```diff
@@ -434,7 +433,10 @@ def generate(self):
 
         self.resources.win_to_unix()
 
-        config_header = self.filter_dot(self.toolchain.get_config_header())
+        config_header = self.toolchain.get_config_header()
+        if config_header:
+            config_header = relpath(config_header, self.resources.file_basepath[config_header])
+        print('Config header: ' + config_header)
```

Comment:
This looks like a debug print.

Reply:
That surely is, and shouldn't be there. I'll fix that.